connector/docs/recyclebin.md

```diff
@@ -38,6 +38,144 @@ Given below is a sample request that can be handled by the emptyRecycleBin opera
+    </soapenv:Header>
+    <soapenv:Body>
+        <emptyRecycleBinResponse>
+            <result>
+                <id>0016F00002S4WaGQAV</id>
+                <success>true</success>
+            </result>
+        </emptyRecycleBinResponse>
+    </soapenv:Body>
+</soapenv:Envelope>
+```
+
+###### Related Salesforce documentation
 
 [https://developer.salesforce.com/docs/atlas.en-us.api.meta/api/sforce_api_calls_emptyrecyclebin.htm](https://developer.salesforce.com/docs/atlas.en-us.api.meta/api/sforce_api_calls_emptyrecyclebin.htm)
+
+### Sample configuration
+
+Following example illustrates how to connect to Salesforce with the init operation and emptyRecycleBin operation.
+
+1. Create a sample proxy as below :
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<proxy xmlns="http://ws.apache.org/ns/synapse"
+       name="salesforce_emptyRecycleBin"
+       startOnLoad="true"
+       statistics="disable"
+       trace="disable"
+       transports="http,https">
+   <target>
+      <inSequence>
+         <property xmlns:ns="wso2.connector.salesforce"
+                   expression="//ns:loginUrl/text()"
+                   name="loginUrl"/>
+         <property xmlns:ns="wso2.connector.salesforce"
+                   expression="//ns:username/text()"
+                   name="username"/>
+         <property xmlns:ns="wso2.connector.salesforce"
+                   expression="//ns:password/text()"
+                   name="password"/>
+         <property xmlns:ns="wso2.connector.salesforce"
+                   expression="//ns:blocking/text()"
+                   name="blocking"/>
+         <property xmlns:ns="wso2.connector.salesforce"
+                   expression="//ns:id/text()"
+                   name="id"/>
+         <property xmlns:ns="wso2.connector.salesforce"
+                   expression="//ns:allOrNone/text()"
+                   name="allOrNone"/>
+         <salesforce.init>
+            <loginUrl>{$ctx:loginUrl}</loginUrl>
+            <username>{$ctx:username}</username>
+            <password>{$ctx:password}</password>
+            <blocking>{$ctx:blocking}</blocking>
+         </salesforce.init>
+         <payloadFactory media-type="xml">
+            <format>
+               <sfdc:sObjects xmlns:sfdc="sfdc">
+                  <sfdc:Ids>$1</sfdc:Ids>
+               </sfdc:sObjects>
+            </format>
+            <args>
+               <arg evaluator="xml" expression="$ctx:id"/>
+            </args>
+         </payloadFactory>
+         <salesforce.emptyRecycleBin>
+            <allOrNone>{$ctx:loginUrl}</allOrNone>
+            <sobjects xmlns:sfdc="sfdc">{//sfdc:sObjects}</sobjects>
+         </salesforce.emptyRecycleBin>
+         <respond/>
+      </inSequence>
+      <outSequence>
+         <send/>
+      </outSequence>
+   </target>
+   <description/>
+</proxy>                                                             
+```
+2. Create a xml file called emptyRecycleBin.xml containing the following xml:
```

Comment:
Create an XML file named emptyRecycleBin.xml and copy the XML configurations given below: